### ISSUE TITLE
Missing credentials in config for lots of parallel requests

### ISSUE THREAD

**Issue by mousavian** [markitx/dynamo-backup-to-s3#50]
I happen to face an issue when using AWS temporary security credentials solution in parallel for many tables which cause `Missing credentials in config` error randomly. 
So I need to set some more [general configuration options](http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html) such as `httpOptions` and `maxRetries.`
( some references: [1](https://github.com/aws/aws-sdk-js/issues/205), [2](https://github.com/aws/aws-sdk-js/pull/448), [3](https://github.com/aws/aws-sdk-js/issues/692))

But since `param` object, [which](https://github.com/markitx/dynamo-backup-to-s3/blob/master/lib/dynamo-backup.js#L38) holds aws config options, is not customisable outside of this library, it is not possible.

--
@dylanlingelbach, do you think we can fork, merge these PRs and work around existing issues if you don't get write access in any near future?

**Comment by dylanlingelbach** [comment]
@mousavian yes, I think we should fork, merge the PRs. I am not sure I will be able to get write access in the near future. I'll coordinate
